validate/files.py

```diff
@@ -1,26 +1,14 @@
 ''' File checking functions '''
 from pathlib import Path
-import os
 import warnings
 
 def check_compressed(path:Path, file_extension:str):
     ''' Check file is compressed '''
-    if not file_extension.endswith('.gz'):
-        warnings.warn(f'Warning: file {path} is not zipped.')
+    compression_extensions = ['.gz']
+    if not any(file_extension.endswith(ext) for ext in compression_extensions):
```

Comment:
Can this be replaced with `if not ext in file_extension: ...`?

Looking at, e.g., [detect_file_type_and_extension](https://github.com/uclahs-cds/public-tool-PipeVal/blob/03df2871ebcc693c68b71f674afecbe4a6aa8611/validate/validate.py#L46) it seems this should work.

```
>>> ext=''
>>> p
PosixPath('/var/log/foo.1.gz')

>>> c
['.gz']

>>> for suffix in p.suffixes[::-1]:
...     ext = suffix.lower() + ext
...     print(ext in c)
True
False
>>>
```

Reply:
We could do it this way but I kept it as is to avoid building up the extension again and just do one check per file extension